src/inputs/TextFieldBase.test.tsx

```diff
@@ -25,4 +26,30 @@ describe(TextFieldBase, () => {
     expect(r.query.test_errorMsg).not.toBeInTheDocument();
     expect(r.query.test_helperText).not.toBeInTheDocument();
   });
+
+  it("handles unfocusedPlaceholder correctly", async () => {
+    // When TextFieldBase is first rendered
+    const r = await render(
+      <TextFieldBase
+        inputProps={{}}
+        unfocusedPlaceholder={"Unfocused placeholder text"}
+        label="Test"
+        errorMsg="Error"
+        helperText="Helper"
+      />,
+    );
+
+    // The unfocused placeholder container is rendered
+    expect(r.test_unfocusedPlaceholderContainer).toBeInTheDocument();
+    // And is visible
+    expect(r.test_unfocusedPlaceholderContainer).not.toHaveStyleRule("position", "absolute");
+
+    // And when we focus the field
+    act(() => {
+      r.test.focus();
+    });
```

Comment:
**nit/question:** Would the `focus()` helper from our `rtl-utils` work for this test?

Reply:
sho does!